tidb_binlog/binlogctl/README.md

```diff
@@ -12,19 +12,20 @@ Usage of binlogctl:
 	-data-dir string
 		meta directory path (default "binlog_position")
 	-node-id string
-		id of node, use to delete some node with operation delete-pump and delete-drainer
+		id of node, used to delete some node with operations delete-pump and delete-drainer
 	-pd-urls string
 		a comma separated list of PD endpoints (default "http://127.0.0.1:2379")
 	-ssl-ca string
-		Path of file that contains list of trusted SSL CAs for connection with cluster components.
+		Path of file that contains the list of trusted SSL CAs for connection with cluster components
 	-ssl-cert string
-		Path of file that contains X509 certificate in PEM format for connection with cluster components.
+		Path of file that contains X509 certificate in PEM format for connection with cluster components
 	-ssl-key string
-		Path of file that contains X509 key in PEM format for connection with cluster components.
+		Path of file that contains X509 key in PEM format for connection with cluster components
 	-time-zone Asia/Shanghai
-		set time zone if you want save time info in savepoint file, for example Asia/Shanghai for CST time, `Local` for local time
+		set time zone if you want to save time info in the savepoint file, for example `Asia`/`Shanghai` for CST time and `Local` for the local time
```

Comment:
`Asia`/`Shanghai`  should changed to `Asia/Shanghai`,  Asia/Shanghai is a value of time zone